configs/droid.kmap

```diff
@@ -22,3 +22,4 @@ keycode 226 = ToggleCallHangup
 keycode 256 = ToggleCallHangup
 keycode 528 = CameraFocus
 keycode 766 = Camera
+keycode 457 = Camera
```

Comment:
Could we add a comment above?
And IIUC @Thaodan's comment "Please format the commit message to be more generic", it can apply to other devices with such button, not only 10 III?

Reply:
> And IIUC @Thaodan's comment "Please format the commit message to be more generic", it can apply to other devices with such button, not only 10 III

Yes that what I tried to say.

Reply:
Added a comment, but this is Xperia 10 III custom value. It might apply to other similar devices, but I have no idea on them. Don't think it makes sense to hide where it's coming from.

Reply:
IMO, for custom key codes, it makes sense to mention the device causing the addition. Such breadcrumbs can be useful for figuring out / resolving possible future conflicts. Conversely, what good could result from leaving out that info?